package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Since you remove the `package-lock.json` file (and now using the `yarn.lock` instead) then I encourage you to add a `.npmrc` file

```sh
$ cat .npmrc
package-lock=false
```

Reply:
Yes, already done :)